### PR TITLE
Mark skills

### PR DESCRIPTION
**Pull request by Alan-Penkar**
Comments added on the mark skill (gameplay/skills/mark.py).

### CODE REVIEW

**Comment by chachmu** [comment]
I think you accidentally included a bunch of cmake stuff

File: soccer/gameplay/plays/restarts/our_free_kick.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ class OurFreeKick(standard_play.StandardPlay):
     BumpKickPower = 0.01
     FullKickPower = 1
     MaxShootingAngle = 80
-    # Untested as of now
+    # Untested as of now - I didn't test this either!!!
```
Comment (jgkamat) [inline soccer/gameplay/plays/restarts/our_free_kick.py]: This comment made my day :)

**Comment by Alan-Penkar** [comment]
well sh**, clearly I'm struggling with git. I tried pulling master from rj and creating a new branch- should I do a hard reset and then reintegrate my changes?

File: soccer/gameplay/plays/restarts/our_free_kick.py (hunk 1 above)
Comment (Alan-Penkar) [reply soccer/gameplay/plays/restarts/our_free_kick.py]: on the one hand - good :), on the other- I didn't mean to include this commit... so there's that

**Comment by JNeiger** [comment]
If it's only a few files, it's probably easier to just do the reset. There probably is a proper way to do this through git itself, but I'm not sure what that would be.

**Pull request closed without merging**